lib/super_good/solidus_taxjar.rb

```diff
@@ -29,6 +29,7 @@ class << self
       attr_accessor :taxable_address_check
       attr_accessor :taxable_order_check
       attr_accessor :test_mode
+      attr_accessor :job_queue
```

Comment:
The rest of these settings are alphabetized. Maybe we should keep doing that.

Reply:
Done!